public/app/features/dashboard/components/TransformationsEditor/TransformationsEditor.tsx

```diff
@@ -230,10 +285,20 @@ class UnThemedTransformationsEditor extends React.PureComponent<TransformationsE
   };
 
   renderTransformsPicker() {
+    const styles = getTransformationsGridStyles(config.theme2);
     const { transformations, search } = this.state;
     let suffix: React.ReactNode = null;
     let xforms = standardTransformersRegistry.list().sort((a, b) => (a.name > b.name ? 1 : b.name > a.name ? -1 : 0));
 
+    if (this.state.selectedFilter !== 'viewAll') {
```

Comment:
lets make a variable for this string constant as well, it's used a few times around

Reply:
done